evtx.go

```diff
@@ -375,6 +375,22 @@ func (self *ParseContext) ConsumeUint32() uint32 {
 	return result
 }
 
+func (self *ParseContext) ConsumeInt32() (ret int32) {
+
+	if len(self.buff) < self.offset+4 {
+		return 0
+	}
+
+	buf := bytes.NewReader(self.buff[self.offset:])
+	err := binary.Read(buf, binary.LittleEndian, &ret)
+	if err != nil {
+		fmt.Println("Binary read for Int32 Failed:", err)
+	}
```

Comment:
```suggestion
	if err != nil {
		return 0
	}
```
looks like a left-over debug statement? same with Real64 routine.

Reply:
yes Thanks!